README.md

```diff
@@ -51,7 +51,7 @@ Next, lets configure the development server. To keep things simple we can put th
 var hypernova = require('hypernova/server');
 
 hypernova({
-  devMode: true,
+  clustering: false,
```

Comment:
I mean when you run things in production you don't want to leave it to a chance and usually your config is very detailed, so defaults not that important. In dev people tend just run things as is and see what's going on, and having clustering there would be just confusing. Even your readme starts with `devMode: true`.

As for us, we're using a pm2 and relying on it's scaling/clustering mechanism. And don't need servers to hog all available cpus, especially if not all of the cores might "available" (e.g. nodejs proxy is running on 4/24 cores).